tools/rawtools/rawtools.xml

```diff
@@ -0,0 +1,279 @@
+        #for $input_raw in $input_raws:
+            #set $input_name = re.sub('[^\w\-\.]', '_',$input_raw.element_identifier.split('/')[-1].replace(".raw", "") + ".raw")
+            ln -s -f '${input_raw}' './raws_folder/${input_name}' &&
+        #end for
+
+        #####################
+        ## Raw Tools ##
+        #####################
+        (rawtools.sh
+
+            ## PARSING AND QUANTIFICATION OPTIONS
+
+            ## If any processing option is enabled in parsing & quantification, "parse" command is mandatory
+            #if $pq_options.pq_conditional.matrix_select  == "yes" or $pq_options.chromatogram_conditional.chromatogram_select == "yes" or $pq_options.mgf_conditional.mgf_select  == "yes" or  $pq_options.metrics_boolean  == "yes":
+                parse
+            #end if
+            --directory ./raws_folder
+
+            ## Generates Matrix file
+            #if $pq_options.pq_conditional.matrix_select == "yes":
+                --parse
+            #end if
+
+            ## Generates quantification information
+            #if $pq_options.pq_conditional.matrix_select == "yes" and $pq_options.pq_conditional.q_conditional.q_select == "yes" :
+                --quant
+                --labelingreagent ${pq_options.pq_conditional.q_conditional.q_labeling_reagent}
+            #end if
+
+            ## Unlabeled quant
+            #if $pq_options.pq_conditional.matrix_select == "yes" and $pq_options.pq_conditional.unlabeledquant_boolean  == "yes":
+                --unlabeledquant
+            #end if
+
+            ## Generates Mgf file
+            #if $pq_options.mgf_conditional.mgf_select == "yes":
+                --writemgf
+                --masscutoff ${pq_options.mgf_conditional.mgf_masscutoff}
+            ## DEPRECATED    --intensitycutoff ${pq_options.mgf_conditional.mgf_intensitycutoff}
+                #if $pq_options.mgf_conditional.mgf_refinement == "yes":
+                    -R
+                #end if
+            #end if
+
+            ## Generates chromatogram file
+            #if $pq_options.chromatogram_conditional.chromatogram_select == "yes":
+                --chro ${pq_options.chromatogram_conditional.chromatogram_type}${pq_options.chromatogram_conditional.chromatogram_order}
+            #end if
+
+            ## Generates metrics file
+            #if $pq_options.metrics_boolean  == "yes":
+                --metrics
+            #end if
+
+            --out \$cwd/output
+
+        2>> $temp_stderr) &&
+
+        cat $temp_stderr 2>&1;
+]]>
+    </command>
+    <inputs>
+        <param format="thermo.raw" name="input_raws" type="data" label="Raw file" multiple="true"
+            help="Select the raw file from history"/>
+        <param name="license_agreement" type="boolean" label="Do you agree to the vendor licenses?" help="This tool uses a proprietary vendor library; to run it you must agree to the vendor license. Read it at http://proteowizard.sourceforge.net/licenses/thermo.html">
```

Comment:
I'm not sure the link is now correct, do you have a better link?

Reply:
That link leads to a license for MSFileReader, but we are using RawFileReader. Here is a better one: https://github.com/kevinkovalchik/RawTools/blob/master/RawFileReaderLicense
I can change it to .md or .txt if needed.